moveit_ros/visualization/robot_state_rviz_plugin/src/robot_state_display.cpp

```diff
@@ -430,7 +418,7 @@ void RobotStateDisplay::calculateOffsetPosition()
   Ogre::Vector3 position;
   Ogre::Quaternion orientation;
 
-  context_->getFrameManager()->getTransform(getRobotModel()->getModelFrame(), ros::Time(0), position, orientation);
+  context_->getFrameManager()->getTransform(getRobotModel()->getModelFrame(), rclcpp::Clock().now(), position, orientation);
```

Comment:
Use rclcpp::Clock().now() as a replacement for ros::time::now(), to replace the ros::time you can just use rclcpp::Time :) .  Please, fix the previous ones.

Reply:
This is `Time(0)`, which intends to get the latest available time available AFAIK. But looking at the function and I guess getting current time matches better the expected behavior. Maybe you propose to use rclcpp::Time::now ? instead of Clock.

Reply:
The returning value from a Clock.now() is a Time yes, but is not the same as declaring it with rclcpp::Time(0). With time(0) you are just passing the time as 0, and with Clock().now() you are just getting the rcl time each second/millisecond. For our previous changes we just replace this kind of changes with the rclcpp::time(), as an example you can take this:

https://github.com/AcutronicRobotics/moveit2/commit/022da7cbbdb161df80e1fb8a9454f3c03f749ce9#diff-7e8d20a58c82e13dc1c7c8369618bcfbR153

Reply:
0fc4f6629d8e7ffadaa13200233f1c483bb1f241